### PR TITLE
feat(sdk): Make bedrock the default

### PR DESCRIPTION
**Pull request by roninjin10**
- Make bedrock default
- We didn't want to make this breaking change prebedrock
- Don't delete any code yet do that in a future pr

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5915** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5915" target="_blank">Graphite</a> 
    * **PR #5919** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5919" target="_blank">Graphite</a> 
      * **PR #5920** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5920" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5920?utm_source=stack-comment).

**Comment by roninjin10** [comment]
DX-77

**Review by hamdiallam — APPROVED** [review]

**Comment by roninjin10** [comment]
tests failing looks like linter wil lfix

File: packages/sdk/src/cross-chain-messenger.ts [hunk 1]
```diff
@@ -143,13 +143,8 @@ export class CrossChainMessenger {
     bridges?: BridgeAdapterData
     bedrock?: boolean
   }) {
-    this.bedrock = opts.bedrock ?? false
+    this.bedrock = opts.bedrock ?? true
```
Comment (roninjin10) [inline packages/sdk/src/cross-chain-messenger.ts:146]:
```suggestion
    this.bedrock = opts.bedrock ?? true
    if (!this.bedrock) {
      console.warn('Legacy non bedrock mode is no longer supported and will no longer work in the next minor release')
    }
```

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
Perhaps this needs a rebase

**Comment by roninjin10** [comment]
meh broke the tests.  Will have to fix for this to land.  If they are prebedrock tests I will delete them

**Comment by roninjin10** [comment]
Instead of deleting tests I made bedrock: true for now. They do currently test relavent code paths that otherwise wouldn't have tests

**Pull request merged**